### DISCUSSION TITLE
How to immediately cancel invoices on paid Teleport Cloud account?

### DISCUSSION THREAD

**Discussion by molinalucas** [gravitational/teleport#35287] [Q&A]
I would like, if posssible, to know if it is possibe to cancel future invoices for a Teleport Cloud paid account that I will not use after the attempt of cancel it. 

![image](https://github.com/gravitational/teleport/assets/48143182/2e1fb763-3609-40ce-a7cc-fd1cb2d1d73c)

**Comment by webvictim** [comment]
Teleport Team is billed in arrears, i.e. you pay for what you have already used afterwards. As such, it's not possible to cancel future invoices.
